cluster/addons/calico-policy-controller/calico-node-daemonset.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       labels:
         k8s-app: calico-node
       annotations:
```

Comment:
Should we delete `annotations` entirely?

Reply:
IMO this is not necessary, but if someone is not happy with this. I'd love to remove this from.

Reply:
I just took a look at all the yaml files, they all have `priorityClass: system-node-critical` to set them to critical.

Reply:
I don't think we should have the key with no values but :man_shrugging: